src/Types.d.ts

```diff
@@ -241,3 +240,14 @@ type UserProvidedConfig = {
   telemetry?: boolean;
   appInfo?: AppInfo;
 };
+type OAuthAuthorizeUrlParams = {
```

Comment:
can we define these in the file where they are used?

Reply:
Done!